src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -143,11 +188,16 @@ public void configMotors() {
 		launcherBottomPIDController.setI(0);
 		launcherBottomPIDController.setD(0);
 		launcherBottomPIDController.setFF(0);
+
+		launcherAngleOneMotor.getEncoder().setPosition(launcherAngleEncoder.getPosition());
+		launcherAngleOneMotor.getEncoder().setPositionConversionFactor(80);
```

Comment:
Did we check with mech to make sure 80 is the correct ratio? From my memory it was 180, but could be misremembering.

Reply:
Would also be good to use constants so we don't forget one if we need to update.

Reply:
Yeah they said 180, whoopsies